lib/galaxy/webapps/galaxy/api/job_files.py

```diff
@@ -99,6 +100,14 @@ def create(self, trans, job_id, payload, **kwargs):
                 upload_store
             ), f"Filename provided by nginx ({file_path}) is not in correct directory ({upload_store})"
             input_file = open(file_path)
+        elif "session_id" in payload:
+            # code stolen from basic.py
+            session_id = payload["session_id"]
+            upload_store = trans.app.config.tus_upload_store or trans.app.config.new_file_path
```

Comment:
This should probably not be mixed into tus_upload_store ? Add a new config option for `tus_job_files_store` or add a subdir `/job_files` ?

Reply:
I was somewhat certain people would say I was over engineering if I did that.